site/class-1-play/LarsMarjew.svg

```diff
@@ -0,0 +1,25 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 10 900 800" height="720" width="800">

```

Comment:
This svg is great, but now your bar chart (which is a pie chart? 🤔) is still wrongly placed.